README.md

```diff
@@ -23,16 +23,18 @@ docker run \
 
 ## Building and running
 
-    git clone https://github.com/prometheus-community/postgres_exporter.git
-    cd postgres_exporter
-    make build
-    ./postgres_exporter <flags>
+    $ mkdir -p $GOPATH/src/github.com/prometheus-community
+    $ cd $GOPATH/src/github.com/prometheus-community
```

Comment:
This is not necessary anymore, and not recommended. The `GOPATH` style is [officially deprecated](https://blog.golang.org/modules2019) (Go 1.13).